deploy/kubernetes/4-collector-config.yaml

```diff
@@ -26,6 +26,23 @@ data:
     sinks:
     - proxyAddress: wavefront-proxy.default.svc.cluster.local:2878
       filters:
+        # Optimized metrics collection to focus collection on default dashboard support.
```

Comment:
`# Optimized metrics collection to omit peripheral metrics.`

Reply:
done